BLSQ/iaso
# IA-3595 Fix profile bulk update bugs

Prevent profile bulk update to raise a wrong error when a value is an empty list instead of `None`.

Related JIRA tickets : [IA-3595](https://bluesquare.atlassian.net/browse/IA-3595)

## Changes

- for _list_ values, ensure the `tasks/create/profilesbulkupdate` API treats `None` as an empty list
- adapt tests to reflect this and use the full API payload in some of them

## How to test

The initial bug was:

- on the user admin page
- select a bunch of users to add them a role
- validate this
- then realize that the users were not updated due to this error: `User without the permission menupermissions.iaso_teams cannot add users to team`

This should be fixed and you should be able to perform this operation without any error.

[IA-3595]: https://bluesquare.atlassian.net/browse/IA-3595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
